app/static/js/script.js

```diff
@@ -87,6 +87,7 @@ $(document).on("ready", function () {
         if($("#picker").val() !== ""){
             imageValue = $("#picker").val();
             $("#preview").css("background-color",imageValue.toString());
+	    $("#picker").val("");
```

Comment:
you probably need 4 more spaces in the front so it has the same number of spaces as in the line above.

Reply:
Use spaces only, not tabs @sagar-kalra. Once you are done, it will be good if you provide a deployed link to review your expected changes.